tests/functional/test_functional.py

```diff
@@ -68,3 +69,22 @@ def test_nextflow(pytester):
     pytester.makefile(ext=".yml", test_nextflow=NEXTFLOW_YAML.read_text())
     result = pytester.runpytest("-v", "--keep-workflow-wd-on-fail")
     assert result.ret == 0
+
+
+@pytest.mark.functional
+def test_bash(pytester):
+    pytester.makefile(ext=".yml", test_bash=BASH_YAML.read_text())
+    pytester.maketxtfile(expected="some\nexpected\noutput")
+    result = pytester.runpytest("-v", "--keep-workflow-wd-on-fail")
+    assert result.ret == 0
+
+
+@pytest.mark.functional
+def test_bash_with_diff(pytester):
+    pytester.makefile(ext=".yml", test_bash=BASH_YAML.read_text())
```

Comment:
Same as my other comment. This should fail. Please add it to test_fail_messages.py

If you cannot incorporate it into the existing structure, simply keep it as a separate test.